### PR TITLE
Point to sample qc definitions in gnomad_meta

### PR DESCRIPTION
**Pull request by mike-w-wilson**

### CODE REVIEW

File: gnomad_qc/v3/sample_qc/README.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+* **sex_imputation_ploidy_cutoffs**: Contains sex chromosome ploidy cutoffs used when determining sex chromosome karyotypes for sex imputation.
+    * **x_ploidy_cutoffs**: Cutoffs for X ploidy.
+        * upper_cutoff_X: Upper cutoff for single X.
+        * lower_cutoff_XX: Lower cutoff for double X.
+        * upper_cutoff_XX: Upper cutoff for double X.
+        * lower_cutoff_XXX: Lower cutoff for triple X. 
+    * **y_ploidy_cutoffs**: Cutoffs for Y ploidy.
+        * lower_cutoff_Y: Lower cutoff for single Y.
+        * upper_cutoff_Y: Upper cutoff for single Y.
+        * lower_cutoff_YY: Lower cutoff for double Y.
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:16]:
```suggestion
    * **x_ploidy_cutoffs**: Cutoffs for X ploidy.
        * **upper_cutoff_X**: Upper cutoff for single X.
        * **lower_cutoff_XX**: Lower cutoff for double X.
        * **upper_cutoff_XX**: Upper cutoff for double X.
        * **lower_cutoff_XXX**: Lower cutoff for triple X. 
    * **y_ploidy_cutoffs**: Cutoffs for Y ploidy.
        * **lower_cutoff_Y**: Lower cutoff for single Y.
        * **upper_cutoff_Y**: Upper cutoff for single Y.
        * **lower_cutoff_YY**: Lower cutoff for double Y.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+    * **prob_nfe**: Random forest probability that the sample is of Non-Finnish European ancestry.
+    * **prob_oth**: Random forest probability that the sample is of Other ancestry.
+    * **prob_sas**: Random forest probability that the sample is of South Asian ancestry.
+    * **training_pop_all**: Sample's project_pop or v2_pop if it is not ‘oth'.
+
+* **sample_filters**: Sample QC filter annotations used for the release.
+    * **sex_aneuploidy**: Whether the sample was flagged for sex aneuploidy.
+    * **insert_size**: Whether the sample was flagged for insert size.
+    * **chimera**: Whether the sample was flagged for chimera.
+    * contamination: Whether the sample was flagged for contamination.
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:278]:
```suggestion
    * **contamination**: Whether the sample was flagged for contamination.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+    * **fail_n_deletion_residual**: Whether the sample failed the number of deletion alternate alleles cutoff based on the metric residuals.
+    * **fail_r_het_hom_var_residual**: Whether the sample failed the Het/HomVar call ratio cutoff based on the metric residuals.
+    * **fail_n_transition_residual**: Whether the sample failed the number of transition alternate alleles.
+    * **fail_n_transversion_residual**: Whether the sample failed the number of transversion alternate alleles.
+
+* **qc_metrics_filters**: Set of all sample QC metrics for which each sample was found to be an outlier after computing sample QC metrics using Hail’s [sample_qc]([https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc)) and regressing out the first 8 ancestry assignment PCs.
+* **relatedness_inference**: Information about the sample’s relatedness to other samples within the callset.
+    * relationships: Samples that have a kinship estimate (kin) > 0.05 determined using Hail’s [pc_relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate) with this sample.
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:306]:
```suggestion
    * **relationships**: Samples that have a kinship estimate (kin) > 0.05 determined using Hail’s [pc_relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate) with this sample.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+* **relatedness_inference_cutoffs**: Contains relatedness inference cutoffs used when running Hail's [pc-relate]([https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate)).
+    * **min_individual_maf**: The minimum individual-specific minor allele frequency used to estimate relatedness for a pair of samples.
+    * **min_emission_kinship**: The minimum kinship coefficient cutoff in the results.
+    * **ibd0_0_max**: The IBD0 cutoff used to determine parent-offspring vs full sibling relationships.
+    * **second_degree_kin_cutoff**: The minimum kinship threshold used for filtering a pair of samples with a second degree relationship.
+    * **first_degree_kin_thresholds**: The first degree kinship threshold used for filtering a pair of samples with a first degree relationship. 
+
+* **outlier_detection_metrics**: Contains the linear regression statistics and cutoffs used for filtering outlier samples based on QC metrics. 
+    * **lms**: Linear regression statistics for QC metrics.
+        * **n_snp**: SNP regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:35]:
```suggestion
        * **n_snp**: SNP regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **r_ti_tv**: Transition/Transversion ratio regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:57]:
```suggestion
        * **r_ti_tv**: Transition/Transversion ratio regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **n_insertion**: Insertion regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:79]:
```suggestion
        * **n_insertion**: Insertion regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **n_deletion**: Deletion regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:90]:
```suggestion
        * **n_deletion**: Deletion regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 8]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **r_het_hom_var**: Het/HomVar regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:101]:
```suggestion
        * **r_het_hom_var**: Het/HomVar regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 9]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **n_transition**: Transition regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:112]:
```suggestion
        * **n_transition**: Transition regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 10]
```diff
@@ -0,0 +1,309 @@
...
+            * **standard_error**: Estimated standard error for each PC.
+            * **t_stat**: t-statistic for each PC.
+            * **p_value**: p-value for each PC.
+            * **multiple_standard_error**: Estimated standard deviation of the random error.
+            * **multiple_r_squared**: Coefficient of determination for nested models.
+            * **adjusted_r_squared**: Adjusted multiple_r_squared taking into account degrees of freedom.
+            * **f_stat**: F-statistic for nested models.
+            * **multiple_p_value**: (p-value for the F-test of nested models.
+            * **n**: Number of samples included in the regression. A sample is included if and only if y, all elements of x, and weight (if set) are non-missing.
+        * **n_tranversion**: Transversion regression statistics
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:123]:
```suggestion
        * **n_tranversion**: Transversion regression statistics.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 11]
```diff
@@ -0,0 +1,309 @@
...
+    * **max_pct_contamination**: Filtering threshold to use for max percent contamination (this is a percent not a proportion, e.g. 5% == 5.00, %5 != 0.05).
+    * **max_pct_chimera**: Filtering threshold to use for max percent chimera (this is a percent not a proportion, e.g. 5% == 5.00, %5 != 0.05).
+    * **min_median_insert_size**: Filtering threshold to use for min median insert size.
+
+**Row annotations**:
+
+* **s**: Unique sample ID
+* **project_meta**: Metadata collected from collaborators or previous versions of gnomAD. The description of annotations in this struct can be found in the [gnomad_meta repo](https://github.com/broadinstitute/gnomad_meta/v3.1/README.md) (for internal use only).
+
+* **subsets**: Subsets of the gnomAD release HT that the sample belongs to
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:198]:
```suggestion
* **subsets**: Subsets of the gnomAD release HT that the sample belongs to.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 12]
```diff
@@ -0,0 +1,309 @@
...
+    * **non_topmed**: Whether the sample was included in the ‘non_topmed’ subset. Subset contains variants from samples that are not the TOPMED cohort. Generated using the negation of the ‘project_meta.topmed’ annotation
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:199]:
```suggestion
    * **non_topmed**: Whether the sample was included in the ‘non_topmed’ subset. Subset contains variants from samples that are not the TOPMED cohort. Generated using the negation of the ‘project_meta.topmed’ annotation.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 13]
```diff
@@ -0,0 +1,309 @@
...
+    * **controls_and_biobanks**: Whether the sample was included in the ‘controls_and_biobanks’ subset. Subset contains variants from samples that are either a control or from a biobank project. Generated matching on ‘case_control’ annotation values of ‘control' or 'biobank'
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:200]:
```suggestion
    * **controls_and_biobanks**: Whether the sample was included in the ‘controls_and_biobanks’ subset. Subset contains variants from samples that are either a control or from a biobank project. Generated matching on ‘case_control’ annotation values of ‘control' or 'biobank'.
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 14]
```diff
@@ -0,0 +1,309 @@
...
+    * **fail_r_het_hom_var_residual**: Whether the sample failed the Het/HomVar call ratio cutoff based on the metric residuals.
+    * **fail_n_transition_residual**: Whether the sample failed the number of transition alternate alleles.
+    * **fail_n_transversion_residual**: Whether the sample failed the number of transversion alternate alleles.
+
+* **qc_metrics_filters**: Set of all sample QC metrics for which each sample was found to be an outlier after computing sample QC metrics using Hail’s [sample_qc]([https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc)) and regressing out the first 8 ancestry assignment PCs.
+* **relatedness_inference**: Information about the sample’s relatedness to other samples within the callset.
+    * relationships: Samples that have a kinship estimate (kin) > 0.05 determined using Hail’s [pc_relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate) with this sample.
+
+* **high_quality**: Whether the sample is considered high quality (no hard filter or qc_metric filter flags)
```
Comment (jkgoodrich) [inline gnomad_qc/v3/sample_qc/README.md:308]:
```suggestion
* **high_quality**: Whether the sample is considered high quality (no hard filter or qc_metric filter flags).
```

**Review by jkgoodrich — CHANGES_REQUESTED** [review]
OK, so after looking over the gnomad_meta PR, you were completely right Mike about splitting it up. I went ahead and did the split and then looked over the new additions. I had some extremely tiny things, but then this is good from my point of view, but can you take a look at the changes I made. Mostly make sure I did the split correctly, look at the first couple lines, and look at the `project_meta` annotation.

File: gnomad_qc/v3/sample_qc/README.md [hunk 15]
```diff
@@ -0,0 +1,309 @@
...
+    * **max_n_singleton**: Filtering threshold to use for the max number of singletons.
+    * **max_r_het_hom_var**: Filtering threshold to use for the max ratio of heterozygotes to alternate homozygotes.
+    * **max_pct_contamination**: Filtering threshold to use for max percent contamination (this is a percent not a proportion, e.g. 5% == 5.00, %5 != 0.05).
+    * **max_pct_chimera**: Filtering threshold to use for max percent chimera (this is a percent not a proportion, e.g. 5% == 5.00, %5 != 0.05).
+    * **min_median_insert_size**: Filtering threshold to use for min median insert size.
+
+**Row annotations**:
+
+* **s**: Unique sample ID
+* **project_meta**: Metadata collected from collaborators or previous versions of gnomAD. The description of annotations in this struct can be found in the [gnomad_meta repo](https://github.com/broadinstitute/gnomad_meta/v3.1/README.md) (for internal use only).
```
Comment (mike-w-wilson) [inline gnomad_qc/v3/sample_qc/README.md:196]:
```suggestion
* **project_meta**: Metadata collected from collaborators or previous versions of gnomAD. The description of annotations in this struct can be found in the [gnomad_meta repo](https://github.com/broadinstitute/gnomad_meta/blob/master/v3.1/README.md) (for internal use only).
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 16]
```diff
@@ -0,0 +1,309 @@
...
+        * **lower_cutoff_Y**: Lower cutoff for single Y.
+        * **upper_cutoff_Y**: Upper cutoff for single Y.
+        * **lower_cutoff_YY**: Lower cutoff for double Y.
+    * **f_stat_cutoff**: F-statistic cutoff to roughly divide 'XX' from 'XY' samples. XX samples are below cutoff and XY are above cutoff. Not used in final ploidy annotation.
+
+ * **population_inference_pca_metrics**: Contains parameters from population assignment.
+    * **min_prob**: The minimum cutoff probability of belonging to a given population at which the population was set.	
+    * **include_unreleasable_samples**: Whether unreleasable samples were included in the ancestry PCA.
+    * **max_mislabeled_training_samples**: The max number of mislabeled training samples that were allowed in population assignment.
+    * **n_pcs**: The number of principal components (PCs) used in [PC-project]([https://github.com/broadinstitute/gnomad_methods/blob/0b660daf23854efa5f6baa233ecbedce40d06fd3/gnomad/sample_qc/ancestry.py#L80](https://github.com/broadinstitute/gnomad_methods/blob/0b660daf23854efa5f6baa233ecbedce40d06fd3/gnomad/sample_qc/ancestry.py#L80)).
```
Comment (mike-w-wilson) [inline gnomad_qc/v3/sample_qc/README.md:23]:
```suggestion
    * **n_pcs**: The number of principal components (PCs) used in [PC-project](https://github.com/broadinstitute/gnomad_methods/blob/b95578056f8051d00dcbb749a10e585d48888f82/gnomad/sample_qc/ancestry.py#L80).
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 17]
```diff
@@ -0,0 +1,309 @@
...
+    * **pop_assignment_iterations**: How many population assignment iterations ran.
+
+* **relatedness_inference_cutoffs**: Contains relatedness inference cutoffs used when running Hail's [pc-relate]([https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate)).
```
Comment (mike-w-wilson) [inline gnomad_qc/v3/sample_qc/README.md:26]:
```suggestion
* **relatedness_inference_cutoffs**: Contains relatedness inference cutoffs used when running Hail's [pc-relate](https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate).
```

File: gnomad_qc/v3/sample_qc/README.md [hunk 18]
```diff
@@ -0,0 +1,309 @@
...
+    * **fail_r_insertion_deletion_residual**: Whether the sample failed the Insertion/Deletion allele ratio cutoff based on the metric residuals.
+    * **fail_n_insertion_residual**: Whether the sample failed the number of insertion alternate alleles cutoff based on the metric residuals.
+    * **fail_n_deletion_residual**: Whether the sample failed the number of deletion alternate alleles cutoff based on the metric residuals.
+    * **fail_r_het_hom_var_residual**: Whether the sample failed the Het/HomVar call ratio cutoff based on the metric residuals.
+    * **fail_n_transition_residual**: Whether the sample failed the number of transition alternate alleles.
+    * **fail_n_transversion_residual**: Whether the sample failed the number of transversion alternate alleles.
+
+* **qc_metrics_filters**: Set of all sample QC metrics for which each sample was found to be an outlier after computing sample QC metrics using Hail’s [sample_qc]([https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc)) and regressing out the first 8 ancestry assignment PCs.
```
Comment (mike-w-wilson) [inline gnomad_qc/v3/sample_qc/README.md:304]:
```suggestion
* **qc_metrics_filters**: Set of all sample QC metrics for which each sample was found to be an outlier after computing sample QC metrics using Hail’s [sample_qc](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.sample_qc) and regressing out the first 8 ancestry assignment PCs.
```

**Review by mike-w-wilson — COMMENTED** [review]
Looks like a few links got duplicated and its preventing the hyperlink from working properly so I just deleted the extra one.

**Pull request merged**